awx/api/serializers.py

```diff
@@ -4563,13 +4573,34 @@ def validate(self, attrs):
                     not_allowed = requested_use_credentials - accessible_use_credentials
                     raise serializers.ValidationError(_(f"Credentials {not_allowed} not found."))
 
+            if requested_use_labels:
+                accessible_use_labels = {tup[0] for tup in Label.objects.all()}
```

Comment:
is there an equivalent `Label.accessible_pk_qs(request.user, 'use_role').all()`?
```suggestion
                accessible_use_labels = {tup[0] for tup in Label.accessible_pk_qs(request.user, 'use_role').all()}
```

Reply:
There is no `Label.accessible_pk_qs()` because accessible_pk_qs is a method from ResourceMixin class which Labels don't extend. Yeah it seems a user/member of an org can see all the labels for that org, similar to what EE does as well. So we need to may be just filter them on the request user org.